src/sections/offering/insurance-coverage.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import { Header, SubTitle } from './price-info'
+import { colors, fonts } from '@hedviginsurance/brand'
+import { Card } from './insured-amount'
+
+interface Props {
+  headline: string
+  subTitle: string
+}
+
+const Container = styled('div')({
+  backgroundColor: colors.OFF_WHITE,
+})
+
+const InnerContainer = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'column',
+})
+
+const BigRow = styled('div')({})
+
+const TitleRow = styled('div')({
+  marginTop: '30px',
+  display: 'flex',
+  alignItems: 'baseline',
+  flexDirection: 'row',
+  '@media (max-width: 700px)': {
+    flexDirection: 'column',
+  },
+})
+
+const Row = styled('div')({
+  marginBottom: '30px',
+  display: 'flex',
+  alignItems: 'baseline',
+  flexDirection: 'row',
+  '@media (max-width: 700px)': {
+    flexFlow: 'wrap',
+    justifyContent: 'flex-start',
+  },
+})
+
+const Col = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'column',
+  marginBottom: '30px',
+})
+
+export const CardHeader = styled('h2')({
+  fontFamily: fonts.MERRIWEATHER,
+  fontSize: '20px',
+  fontWeight: 'normal',
+  maxWidth: '100px',
+  textAlign: 'center',
+  marginBottom: '0px',
+  marginTop: '0px',
+})
+
+const ImageIcon = styled('img')({
+  marginBottom: '10px',
+  marginTop: '10px',
+  marginRight: '30px',
+  marginLeft: '30px',
+  width: '72px',
+  height: '72px',
+})
+
+const Label = styled('p')({
+  marginBottom: '0px',
+  marginTop: '0px',
+  fontSize: '16px',
+  textAlign: 'center',
+  maxWidth: '100px',
+  color: colors.DARK_GRAY,
+})
+
+export class InsuranceCoverage extends React.Component<Props> {
+  constructor(props: any) {
```

Comment:
`any` = 🔪 imo. This should probably be `constructor(props: Props)`, but i don't really see the need for a constructor here :)